src/Spago/GlobalCache.hs

```diff
@@ -100,16 +100,20 @@ getMetadata cacheFlag = do
         Nothing -> mempty
         Just a  -> a
 
-      downloadMeta = do
-        metaBS <- (Http.httpBS metaURL >>= pure . Http.getResponseBody)
-        case decodeStrict' metaBS of
-          Nothing -> do
+      downloadMeta = handleAny
+        (const $ do
```

Comment:
Could we `echoDebug` the exception here? In this way it would only show if one uses `--verbose`

Reply:
Done 😄